app/src/main/java/org/oppia/android/app/onboarding/OnboardingActivity.kt

```diff
@@ -15,7 +15,7 @@ class OnboardingActivity : InjectableAppCompatActivity(), RouteToProfileListList
   companion object {
     fun createOnboardingActivity(context: Context): Intent {
       val intent = Intent(context, OnboardingActivity::class.java)
-      intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
+      intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
```

Comment:
sorry i was not clear in my last comment. As the android docs mention [here](https://developer.android.com/reference/android/content/Intent#FLAG_ACTIVITY_CLEAR_TOP) 

`FLAG_ACTIVITY_CLEAR_TOP` should be used in conjunction with `FLAG_ACTIVITY_NEW_TASK`. so I suggest changing this line to `.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_NEW_TASK)`